src/amo/components/AddonMeta.js

```diff
@@ -16,7 +16,7 @@ export class AddonMetaBase extends React.Component {
     const { addon, i18n } = this.props;
     const averageDailyUsers = addon.average_daily_users;
     const averageRating = addon.ratings.average;
-    const addonReviewCount = addon.ratings.count;
+    const addonRatingCount = addon.ratings.count;
```

Comment:
This was mislabelled, it seems. The API says that is a count of ratings, not reviews. https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#detail